src/applications/vaos/appointment-list/components/PastAppointmentsListV2/index.jsx

```diff
@@ -87,25 +89,28 @@ export function getPastAppointmentDateRangeOptions(today = moment()) {
   return options;
 }
 
-function PastAppointmentsListNew({
-  showScheduleButton,
-  dateRangeOptions = getPastAppointmentDateRangeOptions(),
-  pastAppointmentsByMonth,
-  pastStatus,
-  facilityData,
-  fetchPastAppointments,
-  startNewAppointmentFlow,
-  pastSelectedIndex,
-  hasTypeChanged,
-}) {
+export default function PastAppointmentsListNew() {
+  const dispatch = useDispatch();
   const [isInitialMount, setInitialMount] = useState(true);
+  const dateRangeOptions = getPastAppointmentDateRangeOptions();
+  const {
+    showScheduleButton,
+    pastAppointmentsByMonth,
+    pastStatus,
+    facilityData,
+    pastSelectedIndex,
+    hasTypeChanged,
+  } = useSelector(state => getPastAppointmentListInfo(state));
```

Comment:
We should import and use `shallowEqual` here since this selector function is returning an object.

Reply:
On a related note, I like this pattern of having a single selector method for each component as opposed using multiple `useSelector` calls.

Reply:
Good catch, I did this one from memory and forgot the shallowEqual 🤦